cpp/oneapi/dal/backend/primitives/reduction/test/reduction_rm_cw_perf_dpc.cpp

```diff
@@ -46,8 +46,13 @@ class reduction_rm_test_uniform : public te::float_algo_fixture<std::tuple_eleme
         width_ = GENERATE(16, 128, 1024);
         stride_ = GENERATE(16, 128, 1024);
         height_ = GENERATE(16, 128, 1024, 16384, 32768);
-        SKIP_IF(width_ > stride_);
-        REQUIRE(width_ <= stride_);
+        CAPTURE(width_, stride_, height_);
+    }
+
+    void generate_special() {
```

Comment:
`generate_special` - a meaningless name for me. Can we provide more detailed naming?

Reply:
Tbh I can not imagine more meaningful name for this function